cranelift/filetests/filetests/runtests/simd-fma.clif

```diff
@@ -1,5 +1,7 @@
+test interpret
```

Comment:
`fma` is somewhat broken in the interpreter for the `x86_64-pc-windows-gnu` target (see #4517 for more details). We have a smaller subset of fma tests for the interpreter in a separate file.